### DISCUSSION TITLE
add target_error for gltfpack simplification

### DISCUSSION THREAD

**Discussion by arpu** [zeux/meshoptimizer#657] [Ideas]
gltf-transform https://github.com/donmccurdy/glTF-Transform/blob/ef4e81ac80baaf4f1d95aeb8bd9d5c9c08584cba/packages/functions/src/simplify.ts#L49C4-L49C91

allows to set the target error on simplification
- ratio=0.0, error=0.0001: Aims for maximum simplification, constrained to 0.01% error.

would be nice to have this for gltfpack

**Comment by zeux** [comment]
Yeah this is tracked in #494, it's complicated by the fact that gltfpack currently dynamically selects the error based on the algorithm chosen, but maybe that's fine if we just ignore the error when using aggressive simplification or maybe this can be specified in some other way.

**Comment by arpu** [comment]
at this note would be nice to have the meshopt_simplifyWithAttributes mode too 
3 different simplify modes with  optional error target

**Reply by zeux** [reply]
Yeah that’s tracked in #652
